pylint/checkers/similar.py

```diff
@@ -427,7 +427,10 @@ def reduce_map_data(cls, linter, data):
         """Reduces and recombines data into a format that we can report on
 
         The partner function of get_map_data()"""
-        recombined = SimilarChecker(linter)
+        try:
+            recombined = linter._checkers["similarities"][0]
```

Comment:
As suggested in #4175 a better way of doing this might be to change this method from a `@classmethod` to an instance method. The call-site supports this.